skyportal/models.py

```diff
@@ -362,12 +362,12 @@ class Instrument(Base):
     spectra = relationship('Spectrum', back_populates='instrument')
 
     # can be [] if an instrument is spec only
-    filters = sa.Column(ArrayOfEnum(allowed_bandpasses), nullable=False,
+    filters = sa.Column(ArrayOfEnum(allowed_bandpasses), nullable=True,
```

Comment:
This needs to be nonnull for other handlers (gets set to [] if null)

Reply:
There are examples in the data you sent where there were no filters.

Reply:
Yeah, there they get set to the default (`[]`)

Reply:
Ok, got it. I need to fix some logic in the code.

Reply:
Should this be reverted to `nullable=False`, then?

Reply:
Keeping this as `nullable=False`, given the default value.